core/src/AbilityHelpers.ts

```diff
@@ -169,8 +179,16 @@ class AbilityHelpers implements Types.AbilityHelpersCore, Types.AbilityHelpersIn
     }
 }
 
-export function createAbilityHelpers(win: Window, props?: Types.AbilityHelpersCoreProps): Types.AbilityHelpersCore {
-    return new AbilityHelpers(win, props);
+export function createAbilityHelpers(win: WindowWithAHInstance, props?: Types.AbilityHelpersCoreProps): Types.AbilityHelpersCore {
```

Comment:
That's the public API, it should just have Window in the signature, let's do `win as WindowWithAHInstance` internally in the function.